app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt

```diff
@@ -65,13 +65,6 @@ class ParseSusiResponseHelper {
                 null
             }
 
-            Constant.RSS -> datumList = try {
-                susiResponse.answers[0].data
-            } catch (e: Exception) {
-                Timber.e(e)
-                null
-            }
-
```

Comment:
Is it just this?

Reply:
@iamareebjamal the viewholders and adapters related to this action are also used by other actions so only this part needs to be removed